build/helper/documentation_helper.py

```diff
@@ -195,7 +195,7 @@ def get_docstring_admonition_snippet(node, admonition, d, config, indent=0, extr
 
 def add_attribute_rep_cap_tip_docstring(attr, config):
     '''Add the appropriate (r/w/rw/none) docstring formatted tip for an attribute'''
```

Comment:
Can you please fix this existing mistake - the method docstring is not accurate.